navigator/src/main/java/com/solvd/laba/persistence/CityRepository.java

```diff
@@ -1,4 +1,18 @@
 package com.solvd.laba.persistence;
 
-public class CityRepository {
+import com.solvd.laba.model.City;
+import com.solvd.laba.model.Road;
+
+import java.util.List;
+
+public interface CityRepository {
+
+
+    void create(City city, List<Road> roads);
```

Comment:
I think you should only create a City. Adding list of roads would be in service layer.